src/Components/users/users.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+function Users() {
+  const [results, setResults] = useState([]);
```

Comment:
You are not storing results, you are storing users. Again, be careful how you name variables.